test/main.tf

```diff
@@ -0,0 +1,46 @@
+terraform {
```

Comment:
uuuuum, I think you included this by accident?

Reply:
Yeah